From 6b5db966ae01c0bdaa7b3b90d9332394af6ea0be Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sat, 10 Nov 2012 22:03:43 +0100
Subject: [PATCH] Remove yellow pages support

This support requires yellow pages support in libtirpc, which isn't
available. As most Buildroot users are most likely never going to need
such feature, get rid of it.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 src/security.c |   18 ------------------
 1 file changed, 18 deletions(-)

diff -Naur rpcbind-0.2.3/src/security.c rpcbind-0.2.3.tpg/src/security.c
--- rpcbind-0.2.3/src/security.c	2015-04-27 14:07:43.000000000 +0000
+++ rpcbind-0.2.3.tpg/src/security.c	2015-05-12 20:20:58.729012356 +0000
@@ -24,9 +24,6 @@
 #include <rpcsvc/mount.h>
 #include <rpcsvc/rquota.h>
 #include <rpcsvc/nfs_prot.h>
-#include <rpcsvc/yp.h>
-#include <rpcsvc/ypclnt.h>
-#include <rpcsvc/yppasswd.h>
 #else
 # define MOUNTPROC_MNT	1
 # define MOUNTPROC_UMNT	3
@@ -325,24 +322,9 @@
 		    args->rmt_proc != MOUNTPROC_UMNT)
 			break;
 		goto deny;
-	case YPBINDPROG:
-		if (args->rmt_proc != YPBINDPROC_SETDOM)
-			break;
-		/* FALLTHROUGH */
-	case YPPASSWDPROG:
 	case NFS_PROGRAM:
 	case RQUOTAPROG:
 		goto deny;
-	case YPPROG:
-		switch (args->rmt_proc) {
-		case YPPROC_ALL:
-		case YPPROC_MATCH:
-		case YPPROC_FIRST:
-		case YPPROC_NEXT:
-			goto deny;
-		default:
-			break;
-		}
 	default:
 		break;
 	}
